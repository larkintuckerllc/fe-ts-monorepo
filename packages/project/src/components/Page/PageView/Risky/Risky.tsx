import Fragile from 'COMPONENTS/Fragile';
import React from 'react';

const Risky = () => (
  <div>
    <h2>Risky</h2>
    <Fragile />
  </div>
);
export default Risky;
