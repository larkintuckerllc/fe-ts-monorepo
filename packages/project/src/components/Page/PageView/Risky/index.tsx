import Risky from './Risky';

export default Risky;
