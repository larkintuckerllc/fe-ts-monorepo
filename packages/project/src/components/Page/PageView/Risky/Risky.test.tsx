import Enzyme, { render, shallow } from 'enzyme';
import enzymeAdapterReact16 from 'enzyme-adapter-react-16';
import React from 'react';
import Risky from './Risky';

Enzyme.configure({ adapter: new enzymeAdapterReact16() });

const getDefaultProps = () => ({});

describe('Risky component', () => {
  it('shallow renders without crashing', () => {
    const {} = getDefaultProps();
    shallow(<Risky />);
  });

  it('render snapshot', () => {
    const {} = getDefaultProps();
    const wrapper = render(<Risky />);
    expect(wrapper).toMatchSnapshot();
  });
});
