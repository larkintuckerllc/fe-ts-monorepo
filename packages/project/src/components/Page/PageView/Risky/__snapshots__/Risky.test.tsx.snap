// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Risky component render snapshot 1`] = `
<div>
  <h2>
    Risky
  </h2>
  <div>
    <div
      id="root"
      style="background-color:black"
    >
      Fragile1
    </div>
    <div
      class="my_global"
    >
      Fragile2
    </div>
  </div>
</div>
`;
